src/range-rule.ts

```diff
@@ -63,18 +47,94 @@ export class RangeRule extends ValidationRule {
 			else
 				return rootType.model.getResource("range-at-most").replace("{max}", Property$format(options.property, range.max) || range.max);
 		};
+		
+		options.preExecute = (entity) => {			
+			this["range"] = RangeRule.getRange(entity, options);
+		}
 
 		// call the base type constructor
 		super(rootType, options);
+
+		Object.defineProperty(this, "range", { value: null, writable: true });
```

Comment:
Defined like so to avoid having to move the `super` call to the first line of the constructor